R/lasso-development.R

```diff
@@ -26,6 +24,38 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
+#' @param modelName Optional string. Can specify the model name. If used, you must load the same one in the deploy step.
+#' @section Methods: 
```

Comment:
I agree, that would be nice. I think we should address that with an example overhaul in the future. Rather than have 5 examples for each model, we should have 1 or 2 that cover all the functionality.